extensions/mssql/src/contracts.ts

```diff
@@ -233,4 +262,21 @@ export namespace DeleteAgentProxyRequest {
 	export const type = new RequestType<DeleteAgentProxyParams, sqlops.ResultStatus, void, void>('agent/deleteproxy');
 }
 
+// Job Schedules requests
+export namespace AgentJobSchedulesRequest {
+	export const type = new RequestType<CreateAgentJobScheduleParams, sqlops.AgentJobSchedulesResult, void, void>('agent/schedules');
```

Comment:
Should this be AgentJobScheduleParams instead of CreateAgentJobScheduleParams?

Reply:
thanks!  I'm using this request sucessfully..must be the two `Params` types are "close enough".  I'll fix this.